tests/framework/Optimizers/BayesianOptimizer/Beale/BayesianBeale.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers.BayesianBeale</name>
+    <author>grifanth</author>
+    <created>2023-05-12</created>
+    <classesTested>BayesianOptimizer</classesTested>
+    <description>
+      This is the most basic test of the Optimizer. It optimizes the Beale function using the default settings
+      for the Bayesian Optimizer. This test provides a baseline for functionality and regression testing of the
+      method.
+    </description>
+    <analytic>
+      This test uses Beale's function, which is documented in the analytic tests documentation under
+      the Optimizer functions section.
+    </analytic>
```

Comment:
Is this test an analytic test? If so, try to document it.

Reply:
I believe I removed this test xml as it is not part of the tests added in the PR.